app/Http/Requests/StorePatientRequest.php

```diff
@@ -24,7 +24,13 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'name' => 'required',
+            'email' => 'required',
+            'password' => 'required',
+            'phone' => 'required',
+            'age' => 'required',
+            'address' => 'required',
+            'gender' => 'required',
         ];
     }
```

Comment:
should be!
I want to add messages function() in the following of rules().
sample : 
```
    /**
     * 
     *
     * @return array
     */
    public function messages()
    {
        return [
            'name.required'   => 'Enter a name!',
            'email.required'   => 'Enter an email!',
            'password.required'   => 'Enter password',
            'phone.required'   => 'Enter phone',
            'age.required'   => 'Enter age',
            'address.required'   => 'Enter address',
            'gender.required'   => 'Enter gender',
        ];
    }
```